tests/unit/app/endpoints/test_feedback.py

```diff
@@ -1,17 +1,91 @@
 """Unit tests for feedback endpoint handlers."""
 
+import json
+from pathlib import Path
+
+import pytest
+
 from ols.app.endpoints import feedback
-from ols.app.models.models import FeedbackRequest, FeedbackResponse
-from ols.utils import suid
+from ols.utils import config
+
+
+@pytest.fixture
+def feedback_location(tmpdir):
+    """Fixture sets feedback location to tmpdir and return the path."""
+    config.init_empty_config()
+    config.ols_config.feedback_storage_location = tmpdir.strpath
+    return tmpdir.strpath
+
+
+def store_fake_feedback(path, filename, data):
+    """Store feedback data."""
+    with open(f"{path}/{filename}.json", "w") as f:
+        f.write(json.dumps(data))
```

Comment:
some cleanup mechanism?

Reply:
Are you sure about that? Docs about `tmpdir` fixture says

> Return a temporary directory path object which is unique to each test function invocation...

Reply:
yup, but don't the files remain on FS?

Reply:
if not, I'll LGTM ASAP

Reply:
I believe so.

I've created another test (before the one that lists the dir) that I raised after storing and it seems to be passing.

Reply:
then it looks good, thank you!